README.md

```diff
@@ -24,13 +24,9 @@ By default, Docker Compose use `.env` file as stated here https://docs.docker.co
 
 ## Known Constraint
 
-> This repo is for development only!
+Please be aware that this repository is intended for development purposes only. When copying a folder, such as docker-compose/kafka, and running it with the same command docker compose -f kafka.yaml, it may fail. This is because the container_name is hard-coded in each YAML file. This hard-coding has its advantages, as it allows you to refer to the container by a simple name.
```

Comment:
@MuhammadNurilHuda pls use code short-code if you are mention text that part of the code, such as `docker-compose -f kafka.yaml` or if it part of file path such as `docker-compose/kafka`